download-wgpu-native.py

```diff
@@ -75,7 +76,17 @@ def get_os_string():
 
 
 def get_arch():
-    return "64" if sys.maxsize > 2 ** 32 else "32"  # True on 64-bit Python interpreters
+    # See e.g.: https://stackoverflow.com/questions/45124888
+    is_64_bit = sys.maxsize > 2 ** 32
+    machine = platform.machine()
+    if not is_64_bit:
+        return "i686"
+    elif machine.startswith(("arm", "aarch64")):
+        # Includes Raspberry Pi, MacOS M1, ...
```

Comment:
Maybe quick check with @berendkleinhaneveld that this indeed detects an M1 :)

Reply:
Will check in the morning

Reply:
`arm64`! So we're good to go.